npm/npm-debug-cli/dataplaneConverter/converter.go

```diff
@@ -0,0 +1,300 @@
+package converter
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/Azure/azure-container-networking/npm/npm-debug-cli/dataplaneParser/iptable"
+	"github.com/Azure/azure-container-networking/npm/npm-debug-cli/dataplaneParser/parser"
+	"github.com/Azure/azure-container-networking/npm/npm-debug-cli/pb"
+	"github.com/Azure/azure-container-networking/npm/util"
+	"google.golang.org/protobuf/encoding/protojson"
+)
+
+type Converter struct {
+	ListMap           map[string]string
+	SetMap            map[string]string
+	RequiredChainsMap map[string]bool
+}
+
+func (c *Converter) GetNpmCache(filename ...string) *NPMCache {
+	cachObj := &NPMCache{}
+
+	if len(filename) > 0 {
+		// for dev
+		byteArray, err := ioutil.ReadFile(filename[0])
+		if err != nil {
+			log.Fatalf("Error occured during unmarshalling. Error: %s", err.Error())
+		}
+		err = json.Unmarshal(byteArray, &cachObj)
+		if err != nil {
+			log.Fatalf("Error occured during unmarshalling. Error: %s", err.Error())
+		}
+	} else {
+		// for deployment
+		resp, err := http.Get("http://localhost:10091/npm/v1/debug/manager")
+		if err != nil {
+			log.Fatalf("Error occured during curl. Error: %s", err.Error())
+		}
+		defer resp.Body.Close()
+		byteArray, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			log.Fatalf("Error occured during unmarshalling. Error: %s", err.Error())
+		}
+		err = json.Unmarshal(byteArray, &cachObj)
+		if err != nil {
+			log.Fatalf("Error occured during unmarshalling. Error: %s", err.Error())
+		}
+	}
+
+	return cachObj
+}
+
+// initialize map of chain name that will be include in the result
+func (c *Converter) initConverter(npmCacheFile ...string) {
+	var (
+		npmCache *NPMCache
+	)
```

Comment:
move your declarations as close to the usage of the var as possible, and don't do multi-style unless you have a very good reason for it